kyuubi-common/src/main/scala/org/apache/kyuubi/service/TFrontendService.scala

```diff
@@ -594,6 +594,45 @@ abstract class TFrontendService(name: String)
     resp
   }
 
+  override def UploadData(req: TUploadDataReq): TUploadDataResp = {
+    val resp = new TUploadDataResp()
```

Comment:
the Kyuubi's implementation is not fully same as Hive, let's throw UnsupportedOperationException here directly, then we don't need to modify other classes